frontend/src/metabase/query_builder/components/view/ViewFooter.jsx

```diff
@@ -139,19 +141,23 @@ const ViewFooter = ({
               }
             />
           ),
-          !question.isDataset() && (
-            <EmbedMenu
-              key="embed"
-              resource={question}
-              resourceType="question"
-              hasPublicLink={!!question.publicUUID()}
-              onModalOpen={() =>
-                question.isSaved()
-                  ? onOpenModal("embed")
-                  : onOpenModal("save-question-before-embed")
-              }
-            />
-          ),
+          !question.isDataset() &&
```

Comment:
I believe this is the only single place that conflicts with https://github.com/metabase/metabase/pull/39047 in master. The conflict resolution looks fine to me.